app/mailers/participant_mailer.rb

```diff
@@ -81,6 +83,61 @@ def preterm_reminder(induction_coordinator_profile:)
     ).tag(:preterm_reminder).associate_with(induction_coordinator_profile, as: :induction_coordinator_profile)
   end
 
+  def preterm_reminder_unconfirmed_for_2022(induction_coordinator_profile:)
+    template_mail(
+      PARTICIPANT_TEMPLATES[:preterm_reminder_unconfirmed_for_2022],
+      to: induction_coordinator_profile.user.email,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        name: induction_coordinator_profile.user.full_name,
+        sign_in: new_user_session_url(**UTMService.email(:preterm_reminder_unconfirmed_for_2022)),
+      },
+    ).tag(:preterm_reminder_unconfirmed_for_2022).associate_with(induction_coordinator_profile, as: :induction_coordinator_profile)
+  end
+
+  def fip_register_participants_reminder(induction_coordinator_profile:, school_name:)
+    template_mail(
+      PARTICIPANT_TEMPLATES[:fip_register_participants_reminder],
+      to: induction_coordinator_profile.user.email,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        name: induction_coordinator_profile.user.full_name,
+        school_name:,
+        sign_in: new_user_session_url,
+      },
+    ).tag(:fip_preterm_reminder).associate_with(induction_coordinator_profile, as: :induction_coordinator_profile)
```

Comment:
Do the tag names need updating to the method name or have they been kept the same for a reason?

Reply:
Yep! good spot, thought i'd changed them all. Will do it for both of them